packages/unmutable-core/src/tests/IndexedTests-testUtil.js

```diff
@@ -29,24 +47,15 @@ export default function(test: Function, Wrap: Function, indexedTests: Array<Obje
         tt.deepEqual(Wrap(list).keyArray().value, [0,1,2], 'keyArray returns correct array of keys');
     });
 
+    test('Wrapped Lists return true if empty', (tt: *) => {
```

Comment:
Probably should be a negative case for this test too, just to be safe

Reply:
The negative case already exists in the `CollectionTestDefinitions` file, which is a bit weird that the tests are split like that so I dont mind taking the negative test case out of `CollectionTestDefinitions` and sitting it next to the positive one.